lib/games_collection.rb

```diff
@@ -12,4 +12,59 @@ def generate_objects_from_csv(csv_games_path)
     end
     objects
   end
+
+  # Helper method designed to be reusable; consider moving to a module
+  def every(attribute, collection)
+    collection.map do |element|
+      element.send(attribute)
+    end
+  end
```

Comment:
I like this method but I think we need to  get in the habbit of making our enumberables a one liner so that we save on line space for future much longer methods. so this would look like collection.map {|element| element.send(attribute)} but I also did not know about a  .send method!! that is dope!!

Reply:
Good call. I'll make that change for this and the other ones and push a new commit.